CHANGELOG.md

```diff
@@ -0,0 +1,154 @@
+# Release 0.5.0
+
+This is a significant release, bringing the plugin up to date with the latest
+PennyLane and Qiskit features.
+
+### New features since last release
+
+* The plugin and the tests have been completely re-written from scratch, to ensure
+  high quality and remove technical debt.
+  [#44](https://github.com/XanaduAI/pennylane-qiskit/pull/44)
+  [#43](https://github.com/XanaduAI/pennylane-qiskit/pull/43)
+
+* Samples and variance support have been added to all devices.
+  [#46](https://github.com/XanaduAI/pennylane-qiskit/pull/46)
+
+* Multi-qubit hermitian observables are now supported, due to
+  support being added in Qiskit version 0.12.
+
+* Support has been added for IBM Q Experience 2.0.
+  [#44](https://github.com/XanaduAI/pennylane-qiskit/pull/44)
+
+### Improvements
+
+* Hardware and software devices are now treated identically, with expectations,
+  variance, and samples computed via the _probability_, not the amplitudes.
+  This has several consequences:
+
+  - It makes the code cleaner and simpler, as there is now one defined
+    way of computing statistics.
+
+  - It is faster in most cases, as this does not require computing
+    large matrix-vector products, or Kronecker products. Instead,
+    eigenvalues of single- and multi-qubit observables are computed using
```

Comment:
single (remove -)

Reply:
I think in this case the hyphen is correct, see https://english.stackexchange.com/questions/27707/post-hyphenation-of-split-compound-words